instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/util.py

```diff
@@ -68,3 +68,25 @@ def _format_command_args(args):
         out_str = ""
 
     return out_str
+
+
+def _check_skip(name):
+    skip_list = ["FT.SEARCH", "FT.CREATE", "FT.AGGREGATE"]
```

Comment:
Why are we not tracing these commands in particular?

Reply:
They are traced but with new methods (_traced_search(), etc.). I added that to avoid having repeated spans (one from _traced_execute_command and one from new method). I am not sure if that is the best way to handle it. Maybe we should just keep two spans for those functions or edit current implementation to add those additional attributes to the span?

Reply:
Where is `_traced_search()` defined?

Reply:
Ahh I see, this is an ignore list because these are the commands you are adding more sophisticated tracing for. 

> edit current implementation to add those additional attributes to the span?

In that case I believe this would be the best way to handle this. Utilize the already existing `_traced_execute_command` and create helper methods to determine the span name as well as what attributes should be populated on the span depending on what command it is.